dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh

```diff
@@ -361,33 +387,14 @@ fi
 echo
 echo "[CHE] Deploying Che on ${OPENSHIFT_FLAVOR} (image ${CHE_IMAGE})"
 
-
 DEFAULT_CHE_DEPLOYMENT_FILE_PATH=./che-openshift.yml
 CHE_DEPLOYMENT_FILE_PATH=${CHE_DEPLOYMENT_FILE_PATH:-${DEFAULT_CHE_DEPLOYMENT_FILE_PATH}}
+DEFAULT_CHE_CONFIG_MAP_FILE_PATH="$(pwd)/che-config-map"
```

Comment:
Actually, it is not config map but just env variables list. Maybe it'd better rename it.

Reply:
done

Reply:
done